src/gossip/fuzz.zig

```diff
@@ -122,7 +126,7 @@ pub fn randomPushMessage(rng: std.rand.Random, keypair: *const KeyPair, to_addr:
         &to_addr,
         ChunkType.PushMessage,
     );
-    return packets;
+    return packets.?;
```

Comment:
in this case, it will never be null bc crds_values is not empty (only case it returns null) - but yeah i can see the case for using an empty array - should probs decide what the approach should be going forward